### ISSUE TITLE
KeyError bt_auto_restart

### ISSUE THREAD

**Issue by Ernst79** [custom-components/ble_monitor#695]
```
Logger: root
Source: custom_components/ble_monitor/__init__.py:704
Integration: Passive BLE monitor ([documentation](https://github.com/custom-components/ble_monitor), [issues](https://github.com/custom-components/ble_monitor/issues))
First occurred: 11:52:03 (101 occurrences)
Last logged: 13:32:57

Uncaught thread exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/threading.py", line 973, in _bootstrap_inner
    self.run()
  File "/config/custom_components/ble_monitor/__init__.py", line 704, in run
    if (self.config[CONF_BT_AUTO_RESTART] is True) and (interface_is_ok[hci] is False):
KeyError: 'bt_auto_restart'
```

**Comment by McGiverGim** [comment]
I can confirm, same error here:
```log
2022-02-03 13:33:15 ERROR (Thread-190) [root] Uncaught thread exception
Traceback (most recent call last):
File "/usr/local/lib/python3.9/threading.py", line 973, in _bootstrap_inner
self.run()
File "/config/custom_components/ble_monitor/__init__.py", line 704, in run
if (self.config[CONF_BT_AUTO_RESTART] is True) and (interface_is_ok[hci] is False):
KeyError: 'bt_auto_restart'
```

**Comment by Magalex2x14** [comment]
As far as I can see, the problem is not related to changes in the latest version. Maybe I'm wrong, but in the previous release the same construct was used in the condition. The bt_auto_restart element is missing from self.config. I look further.

**Comment by McGiverGim** [comment]
It seems simply the new "option" is missing in the configuration between versions.
I have fixed simply:
![image](https://user-images.githubusercontent.com/2673520/152346647-01732f7d-5d2a-4615-beb8-8945edc21ee9.png)

In this way it rewrites the config adding the option missing. It seems working now for me.

**Comment by Magalex2x14** [comment]
```python
if (self.config[CONF_BT_AUTO_RESTART] is True) and (interface_is_ok[hci] is False):
```
I think it's worth to revert conditions order:
```python
if (interface_is_ok[hci] is False) and (self.config[CONF_BT_AUTO_RESTART] is True):
```
then `self.config[CONF_BT_AUTO_RESTART]` will be checked less often 😁

**Comment by Ernst79** [comment]
The solution from @McGiverGim works as well. Not sure why it didn't use the default value in the first place, but it is solved. 

Reverting the conditions makes sense, yes

**Comment by Ernst79** [comment]
Added the reversed conditions in #696

**Comment by Magalex2x14** [comment]
Aha ha :) It looks fun for me, sorry :) It's better to tempt fate less often, yes )

**Comment by Ernst79** [comment]
I thought it was serious. 😄

So, what do you want, leave it how it was, or revert it to 

```python
if (interface_is_ok[hci] is False) and (self.config[CONF_BT_AUTO_RESTART] is True):
```

**Comment by Magalex2x14** [comment]
Revert, yes ) It is more efficient, of course... And safer in the context of the error discussed here )

**Comment by Magalex2x14** [comment]
Because the condition to the right of `and` is not checked at all if the left condition is not true.

**Comment by Ernst79** [comment]
Fixed in 7.5.1
